src/adapter.js

```diff
@@ -39,12 +45,13 @@ class VueAdapter extends Adapter {
 		meta = meta || {};
 
 		const renderer = VueServerRenderer.createRenderer();
+		const parsedComponent = this.parseSingleFileVueComponent(str, path);
 
 		const config = this._app.config();
 
-		const vue = new Vue({
+		const vue = new Vue(Object.assign({
 			data: context,
```

Comment:
This causes now problems with the new props. Should be removed and instead something like `renderer.renderToString(vue, context)` should be used - but it's not working as expected